CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## \[Unreleased\]
 
+### Removed
+
+- algo category from algo as it is not required by substra anymore
```

Comment:
Do you plan to remove `substratools.algo.CompositeAlgo` references in an other PR ?